runtimes/eden/src/xcm_config.rs

```diff
@@ -214,6 +215,74 @@ impl orml_xtokens::Config for Runtime {
 	type ReserveProvider = RelativeReserveProvider;
 }
 
+#[cfg(feature = "runtime-benchmarks")]
+parameter_types! {
+	pub const TrustedReserve: Option<(MultiLocation, MultiAsset)> = Some((
+		MultiLocation::here(),
+		MultiAsset{ id: Concrete(Here.into()), fun: Fungible(100) }
+	));
+
+}
+#[cfg(feature = "runtime-benchmarks")]
+impl pallet_xcm_benchmarks::generic::Config for Runtime {
+	type RuntimeCall = RuntimeCall;
+
+	fn worst_case_response() -> (u64, Response) {
+		(0u64, Response::Version(Default::default()))
+	}
+
+	fn transact_origin() -> Result<MultiLocation, BenchmarkError> {
+		Ok(MultiLocation::parent())
+	}
+
+	fn subscribe_origin() -> Result<MultiLocation, BenchmarkError> {
+		Ok(MultiLocation::parent())
+	}
+
+	fn claimable_asset() -> Result<(MultiLocation, MultiLocation, MultiAssets), BenchmarkError> {
+		let origin = MultiLocation::parent();
+		let assets: MultiAssets = (Concrete(MultiLocation::parent()), 1_000u128).into();
+		let ticket = MultiLocation {
+			parents: 0,
+			interior: Here,
+		};
+		Ok((origin, ticket, assets))
+	}
+}
+#[cfg(feature = "runtime-benchmarks")]
+impl pallet_xcm_benchmarks::fungible::Config for Runtime {
+	type TransactAsset = Balances;
+	type CheckedAccount = ();
+	type TrustedTeleporter = ();
+	type TrustedReserve = TrustedReserve;
+	fn get_multi_asset() -> MultiAsset {
+		MultiAsset {
+			id: Concrete(MultiLocation::here()),
```

Comment:
This should be fixed as follows:
```suggestion
			id: Concrete(NodlLocation::get()),
```